code/obj/item/grab.dm

```diff
@@ -660,9 +660,8 @@
 		shot = 1
 
 		if (affecting && assailant && isitem(src.loc))
-			if (get_dist(src.affecting,src.assailant) <= 1)
```

Comment:
Why is this removed? This seems necessary.

Reply:
Distance check is moved to the pointblanking code - it fires a bullet at the target instead of pointblanking if they are more than 1 tile distant. Also, I just realized I need to fix something else. DNM until I get another commit in please

Reply:
Ok, what I thought was an issue (on_pointblank calls) is actually handled in projectile_parent and I don't need to worry about it. nevermind.